package.json

```diff
@@ -75,6 +75,7 @@
     "flagged": "^2.0.1",
     "glider-js": "^1.7.6",
     "history": "^5.0.0",
+    "html-to-image": "github:merapi/html-to-image",
```

Comment:
This library is in active development so forking will add maintenance burden over time. 

Instead, could we work around the video poster issue locally (e.g. preprocess the input DOM tree by replacing video nodes with their poster images) or contribute a hook upstream?

Reply:
^ cc @merapi Since you looked into this.

Reply:
@choumx The author is very slow to respond. We have sent the PRs upstream.
We can fix the poster issue on our side (and other fixes), but that would mean another tree clone (which is rather costly).

Reply:
Another technique is to fork the imported code into a top-level `third_party/` directory in this repo, with a `README` describing the local changes and a `.patch` file that shows the specific diff. This keeps code ownership of the changes within this project (e.g. in case you move off the project in the future). :) 

See an example here: https://github.com/ampproject/amphtml/tree/main/third_party/set-dom

Also, can you create a tracking issue with links to the upstream PRs? We can then use the original repo when the PRs are merged.

Reply:
We use [`patch-package`](https://www.npmjs.com/package/patch-package) to patch third-party libraries like this. Check the `patches` folder for current patches.

Reply:
Even better!

Reply:
Might be better if @merapi does this once he is back from time off on Monday since he's the one who has been working on the `html-to-image` changes.

I'll add it behind a feature flag, merge as-is and Marcin can follow up with the patches when back.

Will add the two issues with links to the upstream PRs meanwhile.

Reply:
Local Issues for tracking:
- https://github.com/google/web-stories-wp/issues/8230
- https://github.com/google/web-stories-wp/issues/8231